src/classes/CW_AudienceCSVProcessHelper.cls

```diff
@@ -0,0 +1,406 @@
+      for(Contact rec: [SELECT id, Account_Name__c FROM Contact WHERE Account_Name__c=:ContactNameList]){
+      	reslt.put(rec.Account_Name__c.ToUpperCase(),rec.id);
+      }
+      return reslt;
+   }
+   
+    private map<string,Id> CreateAudienceGroups(Set<String> GroupNameList ){
+   	list<CW_Audience_Group__c> grpList= new list<CW_Audience_Group__c>();
+      for(String grpName:GroupNameList ){
+     		CW_Audience_Group__c grp = new CW_Audience_Group__c(Campaign__c = CampaignID);
+      	grp.Name=grpName;
+         grp.Is_Control_Group__c = false;
+         grp.Description__c = grpName;
+         grpList.Add(grp);
+      }
+      Insert grpList;
+      map<string,Id> reslt= new map<string,Id>();
+      for(CW_Audience_Group__c rec:grpList ){
+      	reslt.put(rec.Name.ToUpperCase(),rec.id);
+      }
+		return reslt;
+   } 
+   
+   
+   //End OF step 2 - Step 3
+   
+   //Create Campiang Rewards - WORKING OK
+   //Add Error chcking on Insert Records
+   private void loadCreateCampaignReward(){
+   	campRewardMap= new Map<ID,CW_Campaign_Reward__c>();
+   	Map<String,CW_Campaign_Reward__c> campFreeBetdMap= new Map<String,CW_Campaign_Reward__c>();
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		campFreeBetdMap.Put(rec.FreeBetId__c.ToUpperCase(),null);
+   	}
+   	for(CW_Campaign_Reward__c rec: [SELECT Id,Reward_Library__c, Reward_Library__r.FreeBet_Id__c,Reward_Library__r.Amount__c,Reward_Library__r.Currency__c FROM CW_Campaign_Reward__c
+   												 WHERE Reward_Library__r.FreeBet_Id__c=: campFreeBetdMap.keySet()]){ //Campaign__Offer__r.Campaign__c=:CampaignID AND 
+   		campRewardMap.put(rec.Reward_Library__c,rec);								 	
+   		campFreeBetdMap.put(rec.Reward_Library__r.FreeBet_Id__c.ToUpperCase(),rec);
+   												 	
+   	}
+   	List<CW_Campaign_Reward__c> CampRewList= new List<CW_Campaign_Reward__c>();
+   	//For percent we takie cuurency from file
+   	
+   	
+   	for(CW_CSV_Stage_AudienceMembers__c rec:CsvStageList ){
+   		String FreeBet=rec.FreeBetId__c.ToUpperCase();
+   		CW_Campaign_Reward__c rewRec= campFreeBetdMap.get(FreeBet);
+   		if(rewRec==null){
+	   		//rewRec=new CW_Campaign_Reward__c(Campaign_Offer__r.Campaign__c = CampaignID);
+	   		rewRec.Reward_Library__c=rec.Reward_Library__c;
+	   		//rewRec.Reward_Catalogue__c=rec.Reward_Library__r.Reward_Catalogue__c;
+	   		//rewRec.Reward_Curency__c=rec.Reward_Library__r.Currency__c;
+	   		CampRewList.Add(rewRec);
+	   		campFreeBetdMap.put(FreeBet,rewRec);
+   		}
+   	}
+   	insert CampRewList;
+   	for(CW_Campaign_Reward__c rec : CampRewList){
+   		campRewardMap.put(rec.Reward_Library__c,rec);
+   	}
+   }
+   
+   
+   private void  createLoadAudienceGroupSchdule(){
+   	
+   	grupSheduleMap = new Map<ID,CW_Audience_Group_Schedule__c>();//Reference by group ID.
+   	Map<Id,CW_Audience_Group__c> grupMap= new Map<Id,CW_Audience_Group__c>();
+   	CW_Audience_Group_Schedule__c c = new  CW_Audience_Group_Schedule__c();
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.